### DISCUSSION TITLE
Contributions history disappeared after deleting repo

### DISCUSSION THREAD

**Discussion by MartinKratochvilProgramy** [community/community#41356] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

Hi,

I deleted some repositories I wasn't using, but this also deleted relevant contributions in contrubutions history display (my green squares are missing). 

Is this normal? Now it looks like I took large breaks when coding.

**Comment by radar-blip** [comment, marked as answer]
According to the answer given here: https://stackoverflow.com/questions/66104527/does-removing-a-github-repo-also-remove-the-git-contribution-history-from-the-ch
That is normal behaviour.

Also (suggested in the same answer) you could make the repository private, which would keep your contributions.
In your case, you would have to restore your repository first in order to make it private. 

[Restoring a deleted repository](https://docs.github.com/en/repositories/creating-and-managing-repositories/restoring-a-deleted-repository)
